src/patternfly/components/Title/title.scss

```diff
@@ -31,6 +31,18 @@
   --pf-c-title--m-md--FontSize: var(--pf-global--FontSize--md);
```

Comment:
I don't think we'll need these variations on the title component - we should just be able to keep the status colors and treatment in the popover.

Reply:
hi @mcoker would you like me to remove all code I added for title variations (also from Title/examples/Title.md)?

Reply:
@agagancarczyk yep, we shouldn't need any changes to the title component to support this feature.

Reply:
thank you @mcoker , this is fixed now.